wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -483,8 +483,8 @@ data WSettingsApiRecord route = WSettingsApiRecord
   , _localTimeDifference :: route
     :- "time"
     :> "difference"
-    :> Summary "Get local time difference in milliseconds."
-    :> WRes Get Word
+    :> Summary "Get local time difference in microseconds."
+    :> WRes Get Integer
```

Comment:
I've also accidentally spotted that the documentation for the v0 api was incorrectly claiming this were milliseconds. Also, the returned time is now signed, thus the type was switched from `Word` to `Integer`.